src/EnvLoader.php

```diff
@@ -4,34 +4,37 @@
 
 use GuzzleHttp\Client;
 
-class EnvLoader {
-
-    public function envS($name, $default = '') {
-
+class EnvLoader
+{
+    /**
+     * @param string $name
+     * @param mixed  $default
+     *
+     * @return mixed
+     */
+    public function envS($name, $default = null)
+    {
         $options['base_uri'] = getenv('VAULT_ADDR');
         $options['headers']['X-Vault-Token'] = getenv('VAULT_TOKEN');
+        $varsPrefix = getenv('VAULT_VARS_PREFIX');
```

Comment:
i think that this can' be change do we don't have to put it on a environnement variable